modules/textures/docs/README.md

```diff
@@ -21,9 +21,9 @@ npm install @loaders.gl/core
 
 ## API
 
-| Loader                                                            | Description |
-| ----------------------------------------------------------------- | ----------- |
-| [`BasisLoader`](modules/textures/docs/api-reference/basis-loader) |             |
+| Loader                                           | Description |
+| ------------------------------------------------ | ----------- |
+| [`BasisLoader`](docs/api-reference/basis-loader) |             |
```

Comment:
This does not seem like it would work? And seems different from all other links?

Reply:
The link is correct on https://loaders.gl/modules/textures/docs but not on http://ec2-3-130-251-129.us-east-2.compute.amazonaws.com/master/modules/textures/docs (this is a master branch deploy) . I'm not sure where the issue is. Possibly this is the issue of that staging `PATH_PREFIX`, that I use during deploy on AWS.

Reply:
Done